scopes/component/ui/component-meta/component-overview.tsx

```diff
@@ -43,9 +46,18 @@ export function ComponentOverview({
       <div className={textColumn}>
         <div className={styles.componentTitle}>
           <H1>{displayName}</H1>
-          {titleBadges?.map((titleBadge, index) => {
-            return <titleBadge.component key={index} />;
-          })}
+          {titleBadges
+            // @ts-ignore
+            ?.sort((a, b) => a?.weight - b?.weight)
```

Comment:
do no ignore. How about `a?.weigth || 0 - b?.weigth || 0`

Reply:
I wouldn't use 0. it will be confusing, as it's not 0 but unknown.

Reply:
0 will be placed as the first badge. I want it to push it back to the end. which is what happens if it gets `undefined`. its just ts that complains